src/opam_cmd.ml

```diff
@@ -62,8 +62,8 @@ let tag_from_archive archive =
     List.rev path |> List.hd |> tag_of_file ?prefix
   in
   match Uri.scheme uri with
-  | Some "git+http" | Some "git+https" -> Some "master"
-  | Some "git+ssh" -> (
+  | Some "git+http" | Some "git+https"
```

Comment:
I made an unrelated change here. Also, why are github urls treated specially here?

Reply:
It's to determine the tag used for that release. For github URLs, it's possible to write a URL -> tag function, it's not that easy for other URLs.

Reply:
That's actually not entirely true, at this point the tag has already been determined using `tag_from_archive` (which doesn't necessarily rely on `archive` being a github URL) so we should be able to make it work with other URLs.

Reply:
I'll excuse myself, then.

:arrow_right:

Reply:
Hmm ok forget about my comment, I mistook this pattern-matching with the one in `classify_package` so my bad!

Reply:
But this functions has indeed some weird behaviour and the github case is actually never triggered for `git+https` which seems weird.

This is showcased in the tests I added in #6